doc_build/BUILD

```diff
@@ -0,0 +1,109 @@
+# Copyright 2021 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("//:version.bzl", "version")
+load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
+load("@bazel_stardoc//stardoc:stardoc.bzl", "stardoc")
+
+package(
+    default_visibility = ["//visibility:private"],
+)
+
+# HOW TO:
+#   bazel build //doc_build:reference
+#   cp bazel-bin/doc_build/reference.md docs/latest.md
+#   git commit -m 'update docs' docs/latest.md
+
+
+# pairs of rule name and the source file to get it from
+ORDER = [
+    # buildifier: nosort
```

Comment:
Huh.  Does "nosort" even work?  I know "do not sort" does.

FWIW, `buildifier` probably will also munge the formatting here.

Reply:
Changed to buildifier: leave-alone, which is supposed to work. Or, if it doesn't, at least there are open bugs about it. https://github.com/bazelbuild/buildtools/issues/890
Its eems that